chartfx-chart/src/main/java/io/fair_acc/chartfx/axes/spi/AbstractAxisParameter.java

```diff
@@ -816,7 +813,9 @@ public ObjectProperty<AxisLabelOverlapPolicy> overlapPolicyProperty() {
 
     @Override
     public boolean set(final double min, final double max) {
-        return PropUtil.set(minProp, min) | PropUtil.set(maxProp, max);
+        final boolean result = PropUtil.set(minProp, min) | PropUtil.set(maxProp, max);
+        updateCachedTransforms();
```

Comment:
This should be done by the called if needed. Please remove

Reply:
thanks for catching, this was leftover from the other attempt of tackling this. Will change.